packages/shell-api/src/collection.spec.ts

```diff
@@ -33,24 +33,26 @@ import {
   MongoshInvalidInputError,
   MongoshRuntimeError,
 } from '@mongosh/errors';
+import type { StringKey } from './helpers';
 
 const sinonChai = require('sinon-chai'); // weird with import
 
 use(sinonChai);
 describe('Collection', function () {
   describe('help', function () {
-    const apiClass = new Collection({} as any, {} as any, 'name');
+    const apiClass = new CollectionImpl({} as any, {} as any, 'name');
     it('calls help function', async function () {
       expect((await toShellResult(apiClass.help())).type).to.equal('Help');
       expect((await toShellResult(apiClass.help)).type).to.equal('Help');
     });
   });
   describe('signatures', function () {
     it('type', function () {
-      expect(signatures.Collection.type).to.equal('Collection');
+      // TODO: do we want the signatures to be CollectionImpl or Collection?
```

Comment:
Just a reminder for myself to remove this TODO comment before merging and after bringing it up with the team.